app/index.php

```diff
@@ -3,7 +3,19 @@
 require "../vendor/autoload.php";
 $conn = new \App\DbConnector();
 $store = new \App\Store($conn->getDb());
-$books = $store->getAllBooks();
+
+//
+if (!empty($_GET) && $_GET['min'] && $_GET['max']) {
+    $books = $store->getBooksWithinRange($_GET['min'], $_GET['max']);
+    $bookPrices = $store->getAllBookPrices();
+    $sortBooks = new \App\SortBooks($bookPrices);
```

Comment:
Is this an array of sorted books? If so it should be called `$sortedBooks`. At present it sounds like a function name rather than a variable name.